elements/rh-subnav/rh-subnav.ts

```diff
@@ -1,11 +1,11 @@
 import { LitElement, html } from 'lit';
 import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
```

Comment:
`ifDefined` is called but never used. Nuke it.

Reply:
yup orphaned import thought I had deleted.  will clean up.